packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -213,6 +213,15 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({
   const showImageCaptureModal = useCallback(() => {
     const close = showModal('capture-photo-modal', {
       saveFile: (file: UploadedFile) => {
+        if (file.fileType !== 'image') {
+          showSnackbar({
```

Comment:
Because this is a validation error, we should use an inline error notification with low contrast instead of a snack bar. cc @denniskigen am I correct here?

Reply:
Technically, yes. But we shouldn't need to add anything here because validation is already handled in the [MediaUploader](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-attachments-app/src/camera-media-uploader/media-uploader.component.tsx#L14-L15) component.